### DISCUSSION TITLE
3分钟搭建GPT3-QQ机器人！ - Chris's blog

### DISCUSSION THREAD

**Discussion by giscus** [chrisyy2003/chrisyy2003.github.io#14] [评论]
# 3分钟搭建GPT3-QQ机器人！ - Chris's blog

遇到问题？ vscode playwright fastload 有问题可以在下方留言，随后我会把遇到的问题，补充到这里让其他人少踩坑，感谢您的补充。 同时如果您会使用Github也欢迎提Issue。 前言 chatGPT和GPT3的区别 很多小伙伴可能对二者概念有些模糊，其实这是两个不同的东西。 ChatGPT 和 GPT-3 都是由 OpenAI 训练的大型语言模型，但它们有一些关键的区别。G

https://blog.chrisyy.top/qq-gpt3.html#%E6%A1%86%E6%9E%B6

**Comment by Tanem-TN** [comment]
[ERROR] nonebot | Failed to import "nonebot_plugin_gpt3"怎么办呀

**Reply by chrisyy2003** [reply]
什么问题导致的， Failed to import 截下图
